### ISSUE TITLE
ARCTOS hosts not showing scientific name in online interface

### ISSUE THREAD

**Issue by arw36** [globalbioticinteractions/globalbioticinteractions#659]
For input, are there requirements for taxon_names to match a standard? The direct link of arctos URL instead of scientific name likely means looking up by animal taxa's will leave these records omitted. Is best fix to suggest corrections on these? 

Example here: https://www.globalbioticinteractions.org/?interactionType=interactsWith&sourceTaxon=NCBI%3A6211

**Comment by jhpoelen** [comment]
hey Anna @arw36 - thanks for sharing your observation about names with direct links to URL specimen records instead of expected scientific names of the reference records.

For sake of completeness, I've captures a screenshot of an example related to parasite fox tapeworm (_Echinococcus multilocularis_) record http://arctos.database.museum/guid/MSB:Para:21524 and their northern vole (_Microtus oeconomus_) host record at http://arctos.database.museum/guid/MSB:Host:16891 .

https://www.globalbioticinteractions.org/?accordingTo=http%3A%2F%2Farctos.database.museum%2Fguid%2FMSB%3APara%3A21524&interactionType=interactsWith

![Screenshot from 2021-05-03 13-11-50](https://user-images.githubusercontent.com/1084872/116928703-427fe980-ac12-11eb-8409-eeb6051ad756.png)
![Screenshot from 2021-05-03 13-12-22](https://user-images.githubusercontent.com/1084872/116928725-4875ca80-ac12-11eb-983d-5f7cfc326556.png)


The reason why the records sourced from https://github.com/globalbioticinteractions/msb-para aren't getting their names resolves is that the parasite and host records are published in separate archives.

However, if you look in the records from https://github.com/globalbioticinteractions/vertnet , you'll find that the names are showing up as expected. For this dataset, all arctos datasets are included and records reference are resolved and taxonomic information is integrated. 

In short, the unresolved specimen records are a known issue and am hoping to find a good solution for this beyond the existing way as used in https://github.com/globalbioticinteractions/vertnet . See e.g., discussions in #616 and #602  .

@arw36 how would you prefer to define the dependencies of one dataset (e.g., Arctos Parasites) to another (Arctos hosts) so that references can be more easily resolved?

**Comment by jhpoelen** [comment]
see also https://github.com/globalbioticinteractions/globi-taxon-names/issues/7 and https://github.com/globalbioticinteractions/msb-para/issues/1

**Comment by jhpoelen** [comment]
After changes made via https://github.com/globalbioticinteractions/msb-para/commit/a60e2670eebe109893d1d9f336aa4203fc80cf63 , most MSB Para related interaction records are resolved so that to their scientific names can be shown. This includes the example provided in this issue: 

from https://www.globalbioticinteractions.org/?accordingTo=http%3A%2F%2Farctos.database.museum%2Fguid%2FMSB%3APara%3A21524&interactionType=interactsWith see attached screenshots. 

  
![Screenshot from 2021-05-20 11-15-51](https://user-images.githubusercontent.com/1084872/119028844-c8af6600-b95c-11eb-8596-3d55d0a79d3b.png)
![Screenshot from 2021-05-20 11-14-54](https://user-images.githubusercontent.com/1084872/119028853-cb11c000-b95c-11eb-81c3-f1da5ace7297.png)

Also see https://github.com/globalbioticinteractions/msb-para/issues/1#issuecomment-845357452 .

@arw36 please feel free to comments / re-open issue if the issue remains.
